register.lic

```diff
@@ -0,0 +1,76 @@
+=begin
+  Search a deed register for the specified crafting material.
+  Output all lines that match.
+=end
+
+custom_require.call(%w[common common-items drinfomon])
+
+class Register
+  include DRC
+  include DRCI
+
+  def initialize
+    settings = get_settings
+	@search_results = []
+
+    arg_definitions =
+      [
+        [
+          { name: 'query', regex: /^[A-z\s\-\']+$/i, description: 'crafting material to search for, use quotes for multiple words' }
+        ]
+      ]
+
+    args = parse_args(arg_definitions)
+
+    if args.query
+      stow_hands
+	  if get_item('deed register', get_settings.crafting_container)
+	    fput("turn my register to contents")
```

Comment:
Would you be up for making this a `bput`?

Reply:
What's the benefit of bput over fput?

Reply:
`bput` handles round time more reliably